### PR TITLE
added new container classes

### PR DESCRIPTION
**Pull request by joshuafbarker**
Added new container classes.

- Added breakpoints to _variables.scss
- Added container classes to _base.scss
- Added demos to index.html

I also added the box-sizing: border-box fix. It made the alerts look a little different. They still function properly.

### CODE REVIEW

**Comment by cpdevelops** [comment]
Hey there! Before continuing with this pull request, please make sure you have read through the README and CONTRIBUTING files, and have added your changes to the demo site at index.html.

File: index.html [hunk 1]
```diff
@@ -26,6 +26,31 @@
             margin: auto;
           }
         }
+
+        .demo-feature {
```
Comment (cpdevelops) [inline index.html]: What is the purpose of this class? I cannot test right now, but based off the class name it does not seem to do much with no real purpose to the demo site.

Comment (joshuafbarker) [reply index.html]: That is for showing the different containers. That class is for setting a gray box, which holds the blue boxes to show the difference in the containers.

Comment (cpdevelops) [reply index.html]: Ah, ok. Thank you for letting me know. I will go ahead and merge this and make any necessary changes if needed.

Thank you for your contribution and I hope you continue to help out. 

Happy coding!

**Pull request merged**